Sourcery.xcodeproj/project.pbxproj

```diff
@@ -176,7 +184,7 @@
 		098442721E21BF850051E1A1 /* FileParser + AttributesSpec.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = "FileParser + AttributesSpec.swift"; sourceTree = "<group>"; };
 		098442741E2241F10051E1A1 /* Attribute.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = Attribute.swift; sourceTree = "<group>"; };
 		09B1DD381E219D95009228EE /* TypedSpec.stencil */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text; path = TypedSpec.stencil; sourceTree = "<group>"; };
-		09B1DD3A1E219EA3009228EE /* TypedSpec.generated.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = TypedSpec.generated.swift; sourceTree = "<group>"; };
+		09B1DD3A1E219EA3009228EE /* TypedSpec.generated.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = TypedSpec.generated.swift; path = ../../Sourcery/CodeGenerated/TypedSpec.generated.swift; sourceTree = "<group>"; };
```

Comment:
Hm, I didn't notice that it's under ignore... I don't think we should ignore it, so I just removed one that was in tests folder and changed a reference in project to one in generated files folder.
I'm going to start to work on yaml file configuration that you proposed in #143 so we could have at some point a way to set specific destinations for specific templates.